ios/CapacitorCordova.podspec

```
@@ -1,5 +1,11 @@
 require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+prefix = if ENV['NATIVE_PUBLISH']
```

```suggestion
prefix = if ENV['NATIVE_PUBLISH'] == "true"
```
Better check the actual value and not just if it's defined.
Same for the `Capacitor.podspec` file.

And maybe set it to false after publication or call `unset NATIVE_PUBLISH`

Since it's going to be run on the GH actions, the variable will probably not persist executions, but just in case.

---

Also, just to assuage any unease, the only time the podspec is ever evaluated as part of our CI processes is when we're linting and publishing for cocoapods itself. Even if `NATIVE_PUBLISH` is present when publishing to NPM, the podpsec file is not evaluated until a user performs a `pod install` (or in most capacitor users cases, `cap sync`).